odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewMetadata/MetadataItem/MetadataItem.tsx

```diff
@@ -79,31 +87,33 @@ const MetadataItem: React.FC<MetadataItemProps> = ({
   const isCustom = metadataItem.field.origin === 'INTERNAL';
   const isNestedField = (fieldName: string) => fieldName?.indexOf('.') > 0;
 
+  const [hover, setHover] = useState(false);
```

Comment:
Will be better if you will describe hover effects with styled components, you can find an example in DataSourceItemStyles.ts file.

And for the future - please, use react hooks with 
```suggestion
  const [hover, setHover] = React.useState(false);
``` notation, it's our project code style.